res/css/views/elements/_Dropdown.scss

```diff
@@ -41,19 +43,23 @@ limitations under the License.
 }
 
 .mx_Dropdown_arrow {
-    border-color: $primary-fg-color transparent transparent;
-    border-style: solid;
-    border-width: 5px 5px 0;
     display: block;
-    height: 0;
-    position: absolute;
-    right: 10px;
-    top: 14px;
-    width: 0
+    position: relative;
+    width: 10px;
+    height: 6px;
+    padding-right: 8px;
 }
 
-.mx_Dropdown.left_aligned .mx_Dropdown_arrow {
-    left: 10px;
+.mx_Dropdown_arrow::before {
```

Comment:
nit: not sure if you need the psuedo element given mx_Dropdown_arrow is transparent.

Reply:
Ah, perhaps not! I'll take a look again if I am in the area. I think I just got used to setting masks on pseudo-elements... 😅